Source/WebKit/UIProcess/API/C/WKPreferences.cpp

```diff
@@ -157,14 +157,13 @@ bool WKPreferencesGetLoadsImagesAutomatically(WKPreferencesRef preferencesRef)
     return toImpl(preferencesRef)->loadsImagesAutomatically();
 }
 
-void WKPreferencesSetOfflineWebApplicationCacheEnabled(WKPreferencesRef preferencesRef, bool offlineWebApplicationCacheEnabled)
+void WKPreferencesSetOfflineWebApplicationCacheEnabled(WKPreferencesRef, bool)
 {
-    toImpl(preferencesRef)->setOfflineWebApplicationCacheEnabled(offlineWebApplicationCacheEnabled);
 }
 
-bool WKPreferencesGetOfflineWebApplicationCacheEnabled(WKPreferencesRef preferencesRef)
+bool WKPreferencesGetOfflineWebApplicationCacheEnabled(WKPreferencesRef)
 {
-    return toImpl(preferencesRef)->offlineWebApplicationCacheEnabled();
+    return false;
 }
```

Comment:
I think technically these should be moved to the end of the file where the deprecated getters and setters are kept.